Gems/EMotionFX/Code/EMotionFX/Source/MotionEvent.cpp

```diff
@@ -30,6 +30,23 @@ namespace EMotionFX
     {
     }
 
+    MotionEvent::MotionEvent(const MotionEvent& event):
+        Event(event),
+        m_startTime(event.m_startTime),
+        m_endTime(event.m_endTime),
+        m_isSyncEvent(event.m_isSyncEvent)
+    {
+
+    }
+    MotionEvent::MotionEvent(MotionEvent&& event):
+        Event(event),
```

Comment:
update: You need to add `AZStd::move` to this parameter (`event`) otherwise it won't be moved